### ISSUE TITLE
User friendly resource maintenance`

### ISSUE THREAD

**Issue by yawlfoundation** [yawlfoundation/yawl#402]
```
I uploaded new org data in addition to the already existing org data of the 
order fulfilment process.

That gave me a hard time when I was about to set up a new workflow in the 
editor while setting up manual tasks. Due to the amount of resources in the 
wizard it took quite a lot of time to select out the right ones.

As well I wished to have an option to delete a particular set of resources, 
e.g. all resources I uploaded over the order fulfilment org-data file.

Have some ideas. Let's discuss this.
```

Original issue reported on code.google.com by `stephan....@googlemail.com` on 30 Aug 2010 at 12:42

**Comment by yawlfoundation** [comment]
```
See also the last posts of 
http://www.yawlfoundation.org/forum/ldap-authentication-etal
Quite similar issue.
```

Original comment by `neothema...@googlemail.com` on 30 Aug 2010 at 8:25

**Comment by yawlfoundation** [comment]
```
Fixed in editor 3.0 - only shows selections rather than complete list
```

Original comment by `yawl.mic...@gmail.com` on 11 Mar 2014 at 1:24
* Changed state: **Fixed**
